docs/.eleventy/layouts/main.njk

```diff
@@ -5,25 +5,34 @@ description: "Snowpack is the fast, bundle-free build tool for all kinds of fron
 ---
 
 <div class="grid-body-header">
-  <svg height="64px" style="padding-left: 8px;" viewBox="0 0 640 512" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
-      <g id="Page-1" stroke="none" stroke-width="1" fill="currentColor" fill-rule="evenodd">
-          <g id="mountain-solid" transform="translate(-1.000000, 0.000000)"fill-rule="nonzero">
-              <path d="M635.92,462.7 L347.92,14.7 C342.03,5.54 331.89,0 321,0 C310.11,0 299.97,5.54 294.08,14.7 L6.08,462.7 C-0.250773249,472.547007 -0.699487627,485.064987 4.91,495.34 C10.522069,505.612419 21.2945349,512 33,512 L609,512 C620.71,512 631.48,505.61 637.09,495.33 C642.699457,485.058495 642.250708,472.543372 635.92,462.7 Z M321,91.18 L406.39,224 L321,224 L257,288 L218.94,249.94 L321,91.18 Z" id="Shape"></path>
-          </g>
+  <svg height="64px" style="padding-left: 8px;" viewbox="0 0 640 512" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
+    <g id="Page-1" stroke="none" stroke-width="1" fill="currentColor" fill-rule="evenodd">
+      <g id="mountain-solid" transform="translate(-1.000000, 0.000000)" fill-rule="nonzero">
+        <path d="M635.92,462.7 L347.92,14.7 C342.03,5.54 331.89,0 321,0 C310.11,0 299.97,5.54 294.08,14.7 L6.08,462.7 C-0.250773249,472.547007 -0.699487627,485.064987 4.91,495.34 C10.522069,505.612419 21.2945349,512 33,512 L609,512 C620.71,512 631.48,505.61 637.09,495.33 C642.699457,485.058495 642.250708,472.543372 635.92,462.7 Z M321,91.18 L406.39,224 L321,224 L257,288 L218.94,249.94 L321,91.18 Z" id="Shape"></path>
       </g>
+    </g>
   </svg>
   <h1 class="header-snowpack">Snowpack</h1>
   <p class="header-snowpack-subtitle">
     The faster frontend build tool.
   </p>
+  <p class="header-snowpack-blurb">
+    Snowpack is an
+    <u>unbundled</u>
+    front-end development environment that starts in
+    <strong>50 milliseconds or less</strong>. Leveraging the power of native JavaScript ES6 modules, code changes show up instantly in the browser. When it's time to deploy, Snowpack collaborates with your favorite tools (including bundlers) to create a production-ready build.
+    <em>Use a bundler because you want to, not because you need to.</em>
```

Comment:
I really love the opening of this header/blurb! I think it really clarifies some points. But the last 2 sentences feel like they get a bit wordy to me? I’m wondering if we could capture the same with saying “production-ready” somewhere else (maybe earlier)? Sure, that doesn’t communicate as much, but maybe we could link to the `#bundle-for-production` section so users could read more if they’re curious.

Anyways, I’m just saying I think the most important part of the blurb is the beginning, and I’m wondering if the end of the blurb could be shortened somehow.

Reply:
@drwpow I still think it's worth the early call out that unbundled development !== unbundled deployments. I know that you're doing some great work to make unbundled deployments faster in production, but for the moment this is still the biggest misunderstanding / incorrect gut reaction that new readers have.